src/control/server/storage/config.go

```diff
@@ -572,12 +572,126 @@ func parsePCIBusRange(numRange string, bitSize int) (uint8, uint8, error) {
 	return uint8(begin), uint8(end), nil
 }
 
+// AccelProps can be used to select acceleration engine and capabilities.
+// Type is used both in YAML server config and JSON NVMe config files.
+type AccelProps struct {
+	Engine  string   `yaml:"engine,omitempty" json:"accel_engine"`
+	Options []string `yaml:"options,omitempty" json:"-"`
+	OptMask uint16   `yaml:"-" json:"accel_opts"` // Bitmask of optional capabilities.
```

Comment:
As with the earlier comment, I'm not sure why you need both fields on this struct. It seems simpler to just have the `OptMask` field which can either be set directly or set using a helper that takes care of turning the string-based options into a bitmask. You can always reconstruct the string option slice from the bitmask later if that's useful.